vmdb/spec/tools/fix_auth/cli_spec.rb

```diff
@@ -33,10 +33,34 @@
 
   it "should parse hardcoded password" do
     options = described_class.new.parse(%w(-P hardcoded))
-      .options.slice(:hostname, :username, :password, :hardcode, :databases)
+              .options.slice(:hostname, :username, :password, :hardcode, :databases)
     expect(options).to eq(
       :username  => "root",
       :databases => %w(vmdb_production),
       :hardcode  => "hardcoded")
   end
+
+  it "defaults to updating the database" do
+    options = described_class.new.parse(%w())
+              .options.slice(:db, :databaseyml, :key)
+    expect(options).to eq(:db => true)
+  end
+
+  it "doesnt default to database if running another task" do
+    options = described_class.new.parse(%w(--databaseyml))
+              .options.slice(:db, :databaseyml, :key)
+    expect(options).to eq(:databaseyml => true)
+  end
+
+  it "doesnt default to database if running another task 2" do
+    options = described_class.new.parse(%w(--key))
+              .options.slice(:db, :databaseyml, :key)
+    expect(options).to eq(:key => true)
+  end
+
+  it "can run all 3 tasks" do
+    options = described_class.new.parse(%w(--key --db --databaseyml))
+              .options.slice(:db, :databaseyml, :key)
```

Comment:
Minor nitpicks for a follow up PR.  `options` seems like an overloaded name in this context since it appears to be a method name here too.